packages/core/src/core.ts

```diff
@@ -204,3 +214,20 @@ export function saveState(name: string, value: string): void {
 export function getState(name: string): string {
   return process.env[`STATE_${name}`] || ''
 }
+
+/**
+ * Sanatizes an input into a string so it can be passed into issueCommand safely
+ * @param input input to sanitize into a string
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function sanitizeInput(input: any): string {
```

Comment:
have you considered driving this change into command.ts instead? Lowest level would be `escapeData` and `escapeProperty` function to call.

Reply:
Setting process.env in exportVariable makes that hard
https://github.com/actions/toolkit/pull/405/files#diff-396527b4d62db15ebbfb2845654a161fR43

Reply:
Ah good point regarding process.env.

Is command.ts internal only? If so then it should assume type is always string. Otherwise should update types to allow any.